Bottom: 68584f33f54cbbb397008bef050ab1b5a696ebe6
Top:    7fd1fb636076eaf1f309919f2c61effacd2fefcf
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 10:31:28 -0500

Refresh of move-git-checkouts-to

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 6d374c60..e2f4b5e7 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -78,6 +78,7 @@
     remote_src: true
     src: '{{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh'
     dest: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/'
+    mode: 'u=rwx,g=rx,o=rx'
 
 - name: build image
   docker_build:
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 0ca009b7..1d8c4f71 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -75,7 +75,11 @@
     dest: '{{ docker_projects_dir }}/docker-mediawiki'
 
 - name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
+  copy:
+    remote_src: true
+    src: '{{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh'
+    dest: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/'
+    mode: 'u=rwx,g=rx,o=rx'
 
 - name: build image
   docker_build:
